packages/inter-protocol/test/vaultFactory/driver.js

```diff
@@ -288,7 +305,9 @@ const setupServices = async (t, initialPrice, priceBase) => {
       vfPublic,
       aethVaultManager,
     },
-    priceAuthority,
+    priceAuthority: priceAuthorityReg,
+    priceAuthorityAdmin,
+    aethPriceAuthority,
```

Comment:
why expose this? anything looking up aeth prices should be able to get it from `priceAuthority`

OIC, it's to provide access to the manual authority. consider naming it `aethMPA` or something to distinguish.

Reply:
I consider it a bug farm that the thing called `priceAuthority` rather pervasively, is actually a `priceAuthorityRegistry` with the same API as a `priceAuthority`. I don't mind giving the manual PA a distinct name, since one has to know its type to be able to use its manual properties.

Reply:
It's not the shared interface that has my attention. I'm a strong proponent of polymorphism.

What I'm referring to is the confusion of the priceAuthorityRegistry with other priceAuthorities. I've only gradually (over the last two quasi-vacation weeks) come to understand how all these pieces actually fit together.

 * The priceAuthorityRegistry is a vat that pre-dates Zoe in bootstrap order. (It's Vat8, while Zoe is Vat9). That means that it's not a Zoe contract. I don't know what that means for our ability to upgrade it or inform its dependents if it changes.
 * When provided to Auctions and Vaults, its name is PriceAuthority and it is typed as `PriceAuthority`. I had assumed that I was using a PriceAuthority in those contracts.
 
If you have a PriceAuthorityRegistry, and you call `makeQuoteNotifier()`, you end up talking to an ERef for the notifier that refers directly to the intended priceAuthority vat. (This is fine.) If you use any of the other PA methods for requesting a price, every request is relayed through the PriceAuthorityRegistry vat.

ln the context of tracing an associated storage leak (#8400), knowing which vats are holding and relaying which promises has become a priority.

Reply:
> When provided to Auctions and Vaults, its name is PriceAuthority and it is typed as PriceAuthority. I had assumed that I was using a PriceAuthority in those contracts.

Isn't that accurate? It may leave out some details but it is an object that implements PriceAuthority. That's all the Auctions and Vaults are supposed to know.

I'm interested in what changes you might propose. But we're on a tangent now so I'm fine resolving this thread and picking up in Slack or wherever.